docs/product-getting-started-guide/src/main/asciidoc/guided-traffic-rules-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='guided_rules_con']
+= Guided Rules
```

Comment:
I'd just reuse the existing guided-rules-con.adoc in the User Guide. I think it fits just fine. Up to you of course. If you do keep a separate version, then change the anchor ID to use dashes, not underscores.

Reply:
Retained my version as I have a violation screen cap in it.